Models/model.js

```diff
@@ -6,6 +6,16 @@ exports.fetchTopics = () => {
   });
 };
 
+exports.fetchArticles = () => {
+    return db.query(`SELECT author, title, article_id, topic, created_at, votes, COUNT(article_id)::int AS comment_count
```

Comment:
This is for another ticket, but looks like you are not counting the comments, rather the article_ids and naming them comment count - feel free to drop me a message to discuss this one